docs/rst/tutorials/core/qos/mutable/mutable.rst

```diff
@@ -0,0 +1,617 @@
+    * `DomainParticipant QoS <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/domain/domainParticipant/domainParticipant.html?highlight=mutable#domainparticipantqos>`_
+    * `Publisher QoS <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/publisher/publisher/publisher.html?highlight=mutable#publisherqos>`_
+    * `Subscriber QoS <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/subscriber/subscriber/subscriber.html?highlight=mutable#subscriberqos>`_
+
+Another :ref:`tutorial <tutorials_qos_ownership_ownership>` explained the use of the Ownership and Ownership Strength QoS (see `Ownership QoS Policy <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/standardQosPolicies.html#ownershipqospolicy>`_) and how to configure it within the ROS 2 talker/listener demo.
+This tutorial will show how to change Ownership Strength QoS in runtime, after all nodes have been already deployed.
+We will see how three nodes (one subscriber and two publishers) interact: After creation the subscriber will only be receiving data from the publisher with bigger ownership strength (corresponding to exclusive ownership QoS), and after that we will change the ownership strength of the other one to become bigger, thus the subscriber will start to show the data of the latter.
+
+This will be done creating a custom package, following similar steps as in this `ROS 2 tutorial <https://docs.vulcanexus.org/en/latest/ros2_documentation/source/Tutorials/Intermediate/Monitoring-For-Parameter-Changes-CPP.html>`_ to be able to change a node's parameter, and respond to that change by changing the Partition QoS of the publisher.
+
+The ROS MiddleWare (see `ROS 2 RMW <https://docs.vulcanexus.org/en/latest/ros2_documentation/source/Concepts/About-Different-Middleware-Vendors.html?highlight=RMW#default-rmw-implementation>`_) provides APIs to get handles to the objects of the inner DDS implementation.
+That is needed to be able to change the mutable Qos.
+Thus, this tutorial will also show how to use that powerful tool.
+For another demo on how to access inner RMW entities, see `demo_nodes_cpp_native <https://github.com/ros2/demos/tree/master/demo_nodes_cpp_native>`_.
+
+Prerequisites
+-------------
+
+The first prerequisite is to have Vulcanexus Humble installed (see `Linux binary installation <https://docs.vulcanexus.org/en/latest/rst/installation/linux_binary_installation.html>`_ or `Linux installation from sources <https://docs.vulcanexus.org/en/latest/rst/installation/linux_source_installation.html>`_).
+
+Also before starting this tutorial, you should be familiar with creating a workspace and creating a package, as well as familiar with parameters and their function in a ROS 2 system.
+We recommend to first complete the following tutorials:
+
+    * :ref:`Modifying Ownership and Ownership Strength QoS Policy <tutorials_qos_ownership_ownership>`
```

Comment:
I'm not sure this is the most logical order

Reply:
I believe it is: The tutorial for Ownership QoS is done without knowing anything of the ROS 2 packages. It is just using XML configuration ; Then, the rest of tutorials are for creating a package, parameters, using and modifying parameters.